docs/api.md

```diff
@@ -1165,13 +1202,52 @@ console.log(await resultHandle.jsonValue());
 await resultHandle.dispose();
 ```
 
+#### page.exposeBinding(name, playwrightBinding)
```

Comment:
Naming it `exposeBinding` makes it look very different from `exposeFunction` and will likely cause confusion, I'd consider a more similar name `exposeFunctionWithSource` or something, wdyt?

Reply:
Thought about it, but decided to be more neutral - what if we add more data there later...